### PR TITLE
Debug `typescript-effect`

### PR DESCRIPTION
**Pull request by fubhy**

### CODE REVIEW

File: typescript-effect/src/lib.ts [hunk 1]
```diff
@@ -52,7 +31,7 @@ export function program(port: number) {
             yield* scenario1(port),
             yield* scenario2(port),
             yield* scenario3(port),
-            yield* scenario4(port),
+            yield* scenario4(port).pipe(Effect.delay("10 seconds")),
```
Comment (hearnadam) [inline typescript-effect/src/lib.ts:34]: Why is this necessary?

**Review by hearnadam — APPROVED** [review]

File: typescript-effect/src/lib.ts (hunk 1 above)
Comment (jamesward) [reply typescript-effect/src/lib.ts:34]: It is an alternative to dealing with this: https://github.com/jamesward/easyracer/blob/main/javascript-stdlib/lib.js#L62-L66
